install_config/configuring_kuryrsdn.adoc

```diff
@@ -24,115 +24,38 @@ machines (VMs).
 
 Kuryr-Kubernetes and {product-title} integration is primarily designed for
 {product-title} clusters running on OpenStack VMs. Kuryr-Kubernetes components
-are installed as pods on {product-title} in the `openshift-infra` namespace:
+are installed as pods on {product-title} in the `kuryr` namespace:
 
-* kuryr-controller - a single service instance, installed on any node. Modeled
-  in {product-title} as a `Deployment`.
+* kuryr-controller - a single service instance, installed on an `infra` node.
+Modeled in {product-title} as a `Deployment`.
 * kuryr-cni - container installing and configuring Kuryr as CNI driver on each
   {product-title} node. Modeled in {product-title} as a `DaemonSet`.
 
-[[kuryr-sdn-installation]]
-== Installation
-
-The system running openshift-ansible must be subscribed to the OSP as well as
-OCP repositories. The OpenStack integration requires a few extra packages. To
-install the dependencies, run:
-
-----
-$ sudo yum install -y ansible openshift-ansible python2-shade python-dns \
-    python2-heatclient python2-octaviaclient python-openstackclient bind-utils
-----
-
-In the Ansible nodes file, specify the following parameters in order to set up
-Kuryr-Kubernetes as the network plug-in:
-
-----
- # Enable Kuryr.
- openshift_use_openshift_sdn=False
- openshift_use_kuryr=True
- os_sdn_network_plugin_name=cni
-
- # Set userspace so that there are no iptables remains.
- openshift_node_proxy_mode='userspace'
-
- # Keystone URL.
- kuryr_openstack_auth_url=http://127.0.0.1/identity
-
- # OpenStack domain name of user owning Kuryr resources.
- kuryr_openstack_user_domain_name=default
-
- # OpenStack project name of user owning Kuryr resources.
- kuryr_openstack_user_project_name=admin
-
- # OpenStack project id for Kuryr resources.
- kuryr_openstack_project_id=ec0b31802fd043c08bc15b74d2f9a3d3
-
- # OpenStack username that will own kuryr resources.
- kuryr_openstack_username=admin
-
- # Password for that user.
- kuryr_openstack_password=password
-
- # Default Neutron security groups' IDs for Kubernetes pods
- kuryr_openstack_pod_sg_id=f74c83a8-a520-421a-930e-21b6cd098c6a,01f85594-9950-4ded-a92c-5ad546a41188
 
- # Default Neutron subnet ID for Kubernetes pods.
- kuryr_openstack_pod_subnet_id=c85cdee6-0ed1-4d8f-ae61-7afa4674b311
+The Kuryr controller watches the OpenShift API server for pod, service and
+namespace create, update and delete events. It maps the OpenShift API calls to
+corresponding objects in Neutron and Octavia. This means that every network
+solution that implements the Neutron trunk port functionality can be used to
+back OpenShift via Kuryr. This includes open source solutions such as OVS and
```

Comment:
Don't use OpenShift by itself. You can use it as part of an official or official-ish component name, but not by itself. Use {product-title} instead.